src/cljs/athens/self_hosted/client.cljs

```diff
@@ -38,6 +38,8 @@
 
 
 (def ^:private reconnect-timer (atom nil))
+(def ^:private MAX_RECONNECT_TRY 2)
+(def ^:private reconnect-counter (atom -1))
```

Comment:
Nit: surprising initial value, expected it to be 0 or nil. -1 sounds like an artifact of the comparisons rather than the real value of the counter. e.g. if the impl changes this number stops making sense.

Reply:
It's just so I didn't have to do `<=` comparison instead of `<` and allowed me to print amount of reconnects that have happened.

Reply:
@filipesilva are you ok with it staying like this?

Reply:
Yeap am ok. I noticed it and wanted to mention only as a nit (nitpick). Whenever I put nits I mean it as something I noticed but that I think is up to the author.